03/app.js

```diff
@@ -7,12 +7,66 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
+function randomNumber(min, max) {
+    return Math.round((Math.random() * (max - min)) + min);
+}
+
+function getSum(a,b,c) {
+    const numA = parseInt(a);
+    const numB = parseInt(b);
+    const numC = parseInt(c);
+    console.log(numA, numB, numC)
+    let sum = 0;
+
+    if (numA < numB && numA < numC) {
+        sum = numB + numC;
+    }
+    else if (numB < numA && numB < numC) {
+        sum = numA + numC;
+    }
+    else {
+        sum = numA + numB;
+    }
+
+    return sum;
+}
+
+
+const fn = function isEven(num) {
+    if (typeof num !== 'number') {
+            return null;
+        }
+    if (num % 2 === 0){
```

Comment:
W tym przypadku nie ma to znaczenia bo mamy `return`, ale  lepiej jest użyć `else if` bo wtedy nie wykonujemy dodatkowego porównania - mam nadzieje, że jasno się wyraziłem :P